testsuite/test_api.py

```diff
@@ -26,8 +26,8 @@ def setUp(self):
         self._saved_stdout = sys.stdout
         self._saved_stderr = sys.stderr
         self._saved_checks = pycodestyle._checks
-        sys.stdout = PseudoFile()
-        sys.stderr = PseudoFile()
+        sys.stdout = io.StringIO()
+        sys.stderr = io.StringIO()
```

Comment:
FYI, pytest already captures stdout and provides a way to access it: https://docs.pytest.org/en/6.2.x/capture.html#accessing-captured-output-from-a-test-function

Reply:
yeah I'm a pytest core dev

Reply:
it's a larger refactor and this was already large enough